time/src/offset_date_time.rs

```diff
@@ -71,6 +71,67 @@ impl OffsetDateTime {
     }
     // endregion now
 
+    /// Create a new `OffsetDateTime` with the given [[`Date`]], [`Time`], and [`UtcOffset`].
+    ///
+    /// ```
+    /// # use time::{Date, Month, OffsetDateTime, Time, UtcOffset};
+    /// # use time_macros::datetime;
+    /// # fn main() -> Result<(), time::error::Error> {
```

Comment:
This actually isn't necessary, as you can do `Ok::<_, time::Error>(())` at the end.